### DISCUSSION TITLE
Complex styles for element selected?

### DISCUSSION THREAD

**Discussion by bvngee** [davatorium/rofi#1821] [Theming] [answered]
I want to change a bunch of things to the selected element in the listview. For example, change the text padding, add a border, change the fg/bg colors, change the font to bold / bold highlight, etc. But I have only been able to get the fields `text-color`, `background` `border`, and `border-color` to work. Does it have something to do with inheriting values from parents? Or is it just not possible to change more than that? Here is my [config.rasi](https://paste.gg/p/anonymous/d5b073651adb416388a85938caafebb8). I am fairly sure that at least highlight should be possible based on other issue / discussion answers, so maybe it's just something about my config.

**Comment by DaveDavenport** [comment, marked as answer]
What you can change for selected items is limited. It mostly try to avoid that elements/window keeps resizing/moving causing rerenders etc..
I don't think it is good UI to have weirdly resizing element, but  I am not 100% sure what is/isn't possible at the moment but it is limited. (more and more has become possible in newer rofi versions).

For highlighting, please check the rofi-theme manpage on your system, or the online copy on rofi webpage https://davatorium.github.io/rofi/

**Reply by bvngee** [reply]
Ok, I see. Thank you

**Comment by vitordeoliveira** [comment]
use the element-text selected instead

I was having problems to update the selected element, them I tried on the element-text and it works

exemple:

\\ work
element-text selected {
    text-transform: bold lowercase;
  }

\\ dont work
element selected {
    text-transform: bold lowercase;
  }

**Reply by DaveDavenport** [reply]
This is correct,  you need to have the property on  `element-text`, see manpage:
![image](https://github.com/user-attachments/assets/83e5e355-b6d7-4bff-867b-61f20143827d)
